ops/model.py

```diff
@@ -663,3 +663,10 @@ def network_get(self, endpoint_name, relation_id=None):
             if 'relation not found' in str(e):
                 raise RelationNotFoundError() from e
             raise
+
+    def add_metric(self, metrics, labels=None):
+        cmd = ['add-metric']
```

Comment:
Is the command actually add_metric and not add_metrics?   I think add_metrics is more correct since it accepts a set. It's okay for the command itself to be different in this case, since the backend doesn't have to match one-to-one with commands run.

Reply:
Yes, I was under the impression that we should expose `add_metrics` even if `add-metric` is the name of the hook tool but I kept `add_metric` for the backend to be consistent with the tool name but not for the event-level API.

I could have `add_metrics` here as well.